CNV Metrics for WGS

SophieS9 opened the pull request:
Addition of CNV cut offs for WGS data. 
- Average depth > 30X (pass/fail) and total CNV count < 300 (warning only) (See accompanying validation summary). 
- Will display for all runs going forward, but we won't pass/fail on this metric until SOP changes are implemented.
- Unit tests added and all run successfully.

Istaisa commented on config/config_gen01.yaml:
would be useful to add the new metrics to the local config too?

Istaisa commented on pipelines/dragen_pipelines.py:
```diff
@@ -576,4 +576,30 @@ def get_ploidy_metrics(self):
+			cnv_metrics_file = results_path.joinpath(sample).glob(f'*{sample}.cnv_metrics.csv')
+			
+			
+			if len(list(cnv_metrics_file)) == 1:
```
might be better as a try/except to catch silent errors (I'm assuming that the list length should always be 1)

Istaisa commented on qc_database/templates/auto_qc/view_run_analysis.html:
```diff
@@ -702,6 +729,22 @@ <h2> Sample Level QC </h2>
+			{% if sample_analysis.passes_cnv_count %}
+			<td class="table-success">{{ sample_analysis.get_cnv_count }}</td>
+			{% else %}
+			<td class="table-warn">{{ sample_analysis.get_cnv_count }}</td>
```
table-warning not table-warn

Istaisa reviewed (changes requested):
All unit tests pass. Tried with test data, also edited to remove results folder and to get a CNV number warning. All looks good! A couple of minor suggestions but otherwise happy to merge.

SophieS9 commented:
Code review changes
- Added parameters to local config - https://github.com/AWGL/auto_qc/pull/125/commits/35fe97d61d93d1a811cc9ffd3eec49f7ab2d6608 
- For the second request, I've added in an else to return an empty dictionary. https://github.com/AWGL/auto_qc/pull/125/commits/20fc497230e473476e0c4f067272b481008371f4
- Added a couple comments to the models - https://github.com/AWGL/auto_qc/pull/125/commits/49e21e216500697c8981bcb3b14575f7c810d151
- Fixed typo on templates - https://github.com/AWGL/auto_qc/pull/125/commits/497fef709731c137684c4f517d95aff02a4f6c56

Checked upload and still working. 
Unit tests still work after changes.

Istaisa reviewed (approved):
Changes look good - ready to merge to development.
